<?xml version="1.0"?>
<launch>

  <group ns="PF">
    <rosparam file="$(find platform_real)/config/pfParamsReal.yaml" />
  </group>

  <arg name="RobotID"      default="1"/>
  <arg name="stayTime" default="0.25"/>

  <!-- Fake position publisher -->
  <node name="PF" pkg="platform_real" type="infoPFGaden.py" output="screen">
    <param name="stayTime" type="double"  value="$(arg stayTime)"/>
    <param name="RobotID"  type="double"  value="$(arg RobotID)" />
    <param name="printTimeSteps"          value="True"  />
  </node>

  <node name="copter_visualization" pkg="mavros_extras" type="visualizationCustom">
      <param name="num_rotors" value="4"/>
      <param name="arm_len" value="0.22"/>
      <param name="fixed_frame_id" value="map_gaden"/>
      <param name="child_frame_id" value="pid0_frame"/>
  </node>

</launch>
